### DISCUSSION TITLE
Error when trying to update company name

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#254] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.0.2

### Description

Hi!

I am trying to update from CMS the company name of a company. 
Although I am "Company Owner", I get an error saying "Something went wrong on updating item".

From the network console of the browser I can see that the underlying API has responded with 403 status code and the following body: `{"statusCode":403,"error":"Forbidden","message":"Response code 403 (Forbidden)"}`

I am sure I already did this operation on the same company with a previous version of the Console.
Can you help me understanding what is going wrong?

Thanks in advance.

**Comment by davidebianchi** [comment, marked as answer]
Hi @Lucascanna! It is a bug, since it is now possible to change the company configuration only as Console Super User.
We are working on a fix, and we will release it in the next Console release.
